lib/elixir/src/elixir_erl_pass.erl

```diff
@@ -629,3 +634,6 @@ generate_struct_name_guard([{map_field_exact, Ann, {atom, _, '__struct__'} = Key
   {lists:reverse(Acc, [{map_field_exact, Ann, Key, Match} | Rest]), S2};
 generate_struct_name_guard([Field | Rest], Acc, S) ->
   generate_struct_name_guard(Rest, [Field | Acc], S).
+
+format_error(invalid_stacktrace) ->
+  "__STACKTRACE__ is available only inside catch and rescue clauses of the try expression".
```

Comment:
"inside **the** catch and rescue clauses", or "inside catch and rescue clauses of the try expression**s**"?